python-avd/schema_tools/generate_docs/yamlgen.py

```diff
@@ -3,12 +3,16 @@
 # that can be found in the LICENSE file.
 from __future__ import annotations
```

Comment:
Do we need to keep these as we are dropping 3.9?
I thought it was ok starting 3.10 to not have it but a bit confused reagrding python doc and the postponing of the feature: https://docs.python.org/3/library/__future__.html

Reply:
I did not want to change that in this one. The problem is that is has two functionalities. It brings pipe support to <3.9, but it _also_ allows type hints with future refs, meaning a type that is defined later in the file can be given as type hint.